docs/zkapps/experimental.mdx

```diff
@@ -26,7 +26,7 @@ We appreciate any and all feedback you want to provide.
 
 The best place to provide feedback and ask questions is on [Mina Protocol Discord](https://bit.ly/MinaDiscord). 
 
-To ask zkApps questions and engage with other developers building zkApps with SnarkyJS, use the [#zkapps-developers](https://discord.com/channels/484437221055922177/915745847692636181) channel.
+To ask zkApps questions and engage with other developers building zkApps with o1js, use the [#zkapps-developers](https://discord.com/channels/484437221055922177/915745847692636181) channel.
```

Comment:
I learned something new! I searched only on the files that show in the sidebar... you found an instance I had missed, thank you

Reply:
fixed in https://github.com/o1-labs/docs2/pull/861